### PR TITLE
Fix Simulation

### PR DESCRIPTION
**Pull request by jgimeno**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Comment by jgimeno** [comment]
@fedekunze @alessio we are on this fixing the tests, non-determinism is covered I guess now.

File: app/sim_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
+package gaia
```
Comment (alessio) [inline app/sim_test.go]: Mind moving this into a `gaia_test` package please?

File: app/config.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	FlagAllInvariantsValue      bool
+
+	FlagEnabledValue     bool
+	FlagVerboseValue     bool
+	FlagPeriodValue      uint
+	FlagGenesisTimeValue int64
+)
+
+// GetSimulatorFlags gets the values of all the available simulation flags
+func GetSimulatorFlags() {
```
Comment (fedekunze) [inline app/config.go]: no need for all these flags as they are imported from the SDK simapp package

File: app/config.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
+package gaia
```
Comment (fedekunze) [inline app/config.go]: remove this file

File: app/config.go (hunk 2 above)
Comment (jgimeno) [reply app/config.go]: Thanks! Good catch!!

**Review by fedekunze — APPROVED** [review]

**Pull request merged**